### PR TITLE
Enable `include` rule in php-cs-fixer

### PR DESCRIPTION
**Pull request by rob006**
Continuation of #14100
See https://github.com/FriendsOfPHP/PHP-CS-Fixer

Refs https://github.com/yiisoft/yii2-app-basic/issues/123

### CODE REVIEW

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Please resolve conflicts. Thanks.

**Comment by rob006** [comment]
Done.

**Comment by samdark** [comment]
Merged. Thanks!

File: docs/guide/concept-configurations.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ configuration file for the [Basic Project Template](start-installation.md).
 $config = [
     'id' => 'basic',
     'basePath' => dirname(__DIR__),
-    'extensions' => require(__DIR__ . '/../vendor/yiisoft/extensions.php'),
+    'extensions' => require __DIR__ . '/../vendor/yiisoft/extensions.php',
```
Comment (cebe) [inline docs/guide/concept-configurations.md]: Imo the function syntax is easier to read in cases where `require` is used to return something...

Comment (samdark) [inline docs/guide/concept-configurations.md]: It's fine either way. Less characters — less noise.

**Pull request merged**